openslides/motions/access_permissions.py

```diff
@@ -29,7 +29,8 @@ def get_restricted_data(self, full_data, user):
         if user.has_perm('motions.can_see_and_manage_comments') or not full_data.get('comments'):
             data = full_data
         else:
-            data = full_data.copy()
+            from copy import deepcopy
```

Comment:
Sorry. This import has to be at the top of the file. My hint yesterday was a little bit too short.